src/domain/entities/figma-team.ts

```diff
@@ -0,0 +1,17 @@
+export enum FigmaTeamStatus {
```

Comment:
It's the status of a configured team, though it is a confusing at the moment since I've used some of the status values for Figma webhooks (`ACTIVE` and `PAUSED`). I'm thinking of reworking this to just include two statuses, `OK`, `ERROR`.

`OK` - the webhook for the team has been created and the stored OAuth2 credentials for the Figma team admin are valid
`ERROR` - The stored OAuth2 credentials for the Figma team admin are invalid, and the team will need to be re-configured

I don't think we can easily check if a webhook subscription is still active, so the `ERROR` status would just be used to represent invalid OAuth2 credentials.

Reply:
Got it! Thank you for the explanation, Justin!

I like the idea of restricting the enum to just simple 2 values. In addition to this, please, consider renaming the enum to `FigmaTeamAuthStatus` since it describes not the status of the whole team but its auth configuration.

```ts
export enum FigmaTeamAuthStatus {
  OK = 'OK',
  ERROR = 'ERROR',
}
```

If it is difficult to change or you have concerns, please, ignore :)

Reply:
Good suggestion, I've updated the name and squashed the migrations.